### ISSUE TITLE
Collaboration Cycle for [govcio-vebt, 35% Exemption Request from 85/15 Reporting Requirement Application (VA Form 22-10216), VA Form 22-10216]

### ISSUE THREAD

**Issue by arianaadili** [department-of-veterans-affairs/va.gov-team#91665]
### Verify information about your product or feature in the VFS Product Directory.

- [X] I have reviewed the [VFS Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory) and confirm that my information is correct (or that I have taken appropriate actions to update or add my product information to the directory).

### VFS team name

govcio-vebt

### Product name

35% Exemption Request from 85/15 Reporting Requirement Application (VA Form 22-10216)

### Feature name

VA Form 22-10216

### GitHub label for team

vebt

### GitHub label for product

vebt edu forms

### Public DSVA Slack channel for VFS team

https://join.slack.com/share/enQtNzYzNTYyNTY5NTQ2MS00N2E5MDMyMWZhNGIxY2NjYWJiNjEyYTYyNTYzOGZlODAzMDA3YjkyNDUzMjVhOGExMWY2OTBiOTgxNjA0ZjIx

### When did/will you start working on this product/feature?

August 2024

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Tools, applications and dynamic pages

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

No

### Do you need to capture any additional analytics or metrics?

No

### Will a VA editor (Drupal) notice this change?

Unsure

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-education-benefits-tools/Forms/22-10216/collab%20cycle/govcio-vebt_Form_22-10216_Product%20Outline.md

### Verify all relevant materials provided to Governance Team

- [X] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Add the GitHub labels for your team, product, and feature to this ticket.

- [X] I acknowledge that I must add the GitHub labels for my team, product, and feature to this ticket.

**Comment by kristinoletmuskat** [comment]
Hey @arianaadili ! @scjwalker and I will be helping out with this form as well. 

If we have questions about the user flow in mural, who should we tag?

Your form flow is great and super helpful. I don't have a great sense of the overall user journey of where users are currently finding this form, and how it related to the 22-10215. Do you all have that documented anywhere? One though that came to mind looking at both of these forms, is whether they could be combined into one? But I don't have a great sense of like the overall experience and requirements. 

Thank you!
Kristin

**Comment by arianaadili** [comment]
> Hey @arianaadili ! @scjwalker and I will be helping out with this form as well.
> 
> If we have questions about the user flow in mural, who should we tag?
> 
> Your form flow is great and super helpful. I don't have a great sense of the overall user journey of where users are currently finding this form, and how it related to the 22-10215. Do you all have that documented anywhere? One though that came to mind looking at both of these forms, is whether they could be combined into one? But I don't have a great sense of like the overall experience and requirements.
> 
> Thank you! Kristin

Hey @kristinoletmuskat :) You can tag @ameliahackingVA for any questions about user flows. 

This is a non-Veteran facing form, intended only for SCOs to fill out. It will be available on the Resources for Schools page. The forms are related in that the SCO will need to fill out form 10216 if they want to be exempt from the 85/15 reporting requirement. However, they’ll need to submit _both_ forms 10215 and 10216 if their institution is non-accredited and they also want to be exempt. Ultimately, the SCO will upload the form to the EDU portal, which is outlined in the user flow and designs, but we can go into more detail during our design intent meeting.

We have discussed combining the forms, as you suggested, but our stakeholder prefers not to go that route. @ameliahackingVA if I missed anything or if you’d like to add any additional insights, please feel free to jump in.

I hope this answers your question(s)!

**Comment by shiragoodman** [comment]
@alexisbetz I added the Architecture Intent touchpoint info to the ticket per your scheduling. Please lmk if you have any questions.

**Comment by alexisbetz** [comment]
> [@alexisbetz](https://github.com/alexisbetz) Please add the completed Architecture Intent checklist as soon as you can so the team has two business days to review it before the Arch Intent meeting. FYI [@shiragoodman](https://github.com/shiragoodman)

Apologies, we were not able to complete this this today. We will need to reschedule

**Comment by shiragoodman** [comment]
@alexisbetz please follow the guidance in your Architecture Intent event for cancelling and rescheduling. Thank you.

**Comment by DonMcCaugheyUSDS** [comment]
Thanks for participating in today's architecture review meeting.  Your project is in good shape and ready to move to staging review.

Please consider adding some automated tests for your application (if you don't have any) to supplement your manual test plan.

**Comment by shiragoodman** [comment]
@VishnhavAshok yes, it's your teams responsibility to close it out. In the future, please keep in mind that you will need a Staging Review before pushing to prod. Staging Reviews are required.

**Comment by shiragoodman** [comment]
@VishnhavAshok thank you for the clarification! I did not look for that detail.. I just assumed since you had a DI and MPR that they were vet-facing. We don't hold DI's or MPRs for non-Veteran facing work (anymore). Apologies!
